cargo-dist/src/backend/templates.rs

```diff
@@ -189,10 +200,31 @@ impl Templates {
         self.render_file_to_clean_string_inner(file, val)
     }
 
+    /// Render a maybe-templated file to a string, cleaning all newlines to be unix-y
     fn render_file_to_clean_string_inner(
         &self,
         file: &TemplateFile,
         val: &impl Serialize,
+    ) -> DistResult<String> {
+        if file.env == ENV_NONE {
+            self.render_raw_file_to_clean_string(file)
+        } else {
+            self.render_templated_file_to_clean_string(file, val)
+        }
+    }
+
+    /// Render a non-jinja template file to a string, cleaning all newlines to be unix-y
```

Comment:
"Render" kind of implies actually inlining values into the file, might be worth having the comment clarify it's just copying it and redoing newlines?